src/containers/myEvents/ducks/types.ts

```diff
@@ -0,0 +1,31 @@
+import { AsyncRequest } from '../../../utils/asyncRequest';
+import { C4CState } from '../../../store';
+import { ThunkAction } from 'redux-thunk';
+import { MyEventsActions } from './actions';
+import { ProtectedApiExtraArgs } from '../../../api/protectedApiClient';
+
+export interface MyEventsReducerState {
+  readonly myEvents: AsyncRequest<MyEventProps[], any>;
+}
+
+export type MyEventsThunkAction<R> = ThunkAction<
+  R,
+  C4CState,
+  ProtectedApiExtraArgs,
+  MyEventsActions
+>;
+
+export interface MyEventProps {
```

Comment:
If we can reuse or extend EventProps we probably should

Reply:
Agree, I think we should be able to extend `EventProps` and add additional fields. Although, I don't think there are any additional fields there - doesn't the API return how many tickets the user has?